### ISSUE TITLE
[Feature request] Monthly / Yearly graphs

### ISSUE THREAD

**Issue by Onepamopa** [AdguardTeam/AdGuardHome#750]
It would be nice if we can get graphs & statistics (total Nr of DNS requests, blocked by filters, malware/phishing, adult).

Or, at least support for influxdb/grafana at the backend to visualize that data.

**Comment by rashidjehangir** [comment]
Yes - would be amazing to have daily, weekly, monthly and yearly
 data that can be saved and viewed.

**Comment by ArchiveBase** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/776#issuecomment-499468432

**Comment by ArchiveBase** [comment]
I thought to elaborate this feature request instead open new issue.

As @rashidjehangir mentioned it's nice to have daily, weekly, monthly and yearly but i want it extended as below 

1) **Custom time frames**
Ex: From June 2 to June 8

2) **With multiple clients**
Ex: From June 2 to June 8 :: 192.168.0.105 and 192.168.8.120

Regarding view logs under custom time frame has on https://www.glasswire.com and even more various sort ways https://www.ntop.org/products/traffic-analysis/ntop
